modules/app/components/layout/Footer/NewFooter.tsx

```diff
@@ -0,0 +1,235 @@
+          title: t('Gov Tracking Sheet')
+        },
+        {
+          url: 'https://manual.makerdao.com/governance/governance-cycle/monthly-governance-cycle',
+          title: t('Monthly Gov Cycle')
+        },
+        {
+          url: 'https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle',
+          title: t('Weekly Gov Cycle')
+        }
+      ]
+    },
+    {
+      header: t('Products & Tools'),
+      list: [
+        {
+          url: 'https://makerdao.statuspage.io/',
+          title: t('Service Status')
+        },
+        {
+          url: 'https://oasis.app/',
+          title: t('Oasis')
+        },
+        {
+          url: 'https://auctions.makerdao.network/',
+          title: t('Auctions Dashboard')
+        },
+        {
+          url: 'https://migrate.makerdao.com/',
+          title: t('Migrate Dashboard')
+        },
+        {
+          url: 'https://makerburn.com/',
+          title: t('MakerBurn')
+        },
+        {
+          url: 'https://daistats.com/',
+          title: t('DAI Stats')
+        }
+      ]
+    },
+    {
+      header: t('Developer'),
+      list: [
+        {
+          url: 'https://makerdao.com/whitepaper',
+          title: t('Whitepaper')
+        },
+        {
+          url: 'https://docs.makerdao.com/',
+          title: t('Technical Docs')
+        },
+        {
+          url: 'https://vote.makerdao.com/api-docs',
+          title: 'API Docs'
+        },
+        {
+          url: 'https://github.com/makerdao/developerguides',
+          title: 'Developer Guides'
+        },
+        {
+          url: 'https://www.notion.so/makerdao/Maker-Brand-ac517c82ff9a43089d0db5bb2ee045a4',
+          title: t('Brand Assets')
+        },
+        {
+          url: 'https://makerdao.com/en/feeds/',
+          title: t('Oracle Feeds')
+        }
+      ]
+    }
+  ];
+
+  const logos = {
+    makerdao: [
+      { title: 'Discord', url: 'https://chat.makerdao.com', icon: 'discord' },
+      { title: 'Twitter', url: 'https://twitter.com/MakerDAO', icon: 'twitter' },
+      { title: 'Reddit', url: 'https://www.reddit.com/r/MakerDAO/', icon: 'reddit' },
+      { title: 'Telegram', url: 'https://t.me/makerdaoOfficial', icon: 'telegram' },
+      { title: 'YouTube', url: 'https://www.youtube.com/MakerDAO', icon: 'youtube' },
+      { title: 'GitHub', url: 'https://www.github.com/makerdao', icon: 'github' }
+    ],
+    makerdux: [
+      { title: 'Discord', url: 'https://discord.gg/GHcFMdKden', icon: 'discord' },
+      { title: 'Twitter', url: 'https://twitter.com/MakerDUX', icon: 'twitter' },
+      { title: 'GitHub', url: 'https://github.com/makerdao/governance-portal-v2', icon: 'github' },
+      { title: 'Canny', url: 'https://makergovernance.canny.io/', icon: 'canny' },
+      { title: 'Immunifi', url: 'https://immunefi.com/bounty/makerdao/', icon: 'immunifi' }
+    ]
+  };
```

Comment:
This looks possibly duplicated in the `LandingFooter` component. Perhaps we can have a common file to help with all the links and logos?